frontend/src/queries/nodes/InsightQuery/utils/filtersToQueryNode.ts

```diff
@@ -127,6 +135,29 @@ export const filtersToQueryNode = (filters: Partial<FilterType>): InsightQueryNo
 
     // breakdown
     if (isInsightQueryWithBreakdown(query)) {
+        // handle multi-breakdowns
+        if (filters.breakdowns != null) {
+            if (filters.breakdowns.length === 1) {
+                filters.breakdown_type = filters.breakdowns[0].type
+                filters.breakdown = filters.breakdowns[0].property as string
+            } else {
+                captureException(
+                    'Could not convert multi-breakdown property `breakdowns` - found more than one breakdown'
+                )
+            }
+        }
+
+        // handle missing breakdown_type
+        if (filters.breakdown != null && filters.breakdown_type == null) {
```

Comment:
`!=` or `!==`? Here and on line 139... 🤔

Reply:
`!=` since that checks for `undefined` and `null`, but doesn't treat `0` as falsy